1.8/administration/overlay-networks/ip-per-container.md

```diff
@@ -4,30 +4,28 @@ post_title: Configuring IP-per-Container in Overlay Networks
 menu_order: 10
 ---
 
-In Enterprise DC/OS the overlay network feature is enabled by default. The default configuration provides two overlay networks `dcos-1` and `dcos-2`, whose YAML configuration is as follows:
+The overlay network feature is enabled by default in DC/OS. The default configuration provides an overlay network, `dcos`, whose YAML configuration is as follows:
 
 ```yaml
   dcos_overlay_network:
-    vtep_subnet: 172.16/12
+    vtep_subnet: 44.128.0.0/20 
     vtep_mac_oui: 70:B3:D5:00:00:00
     overlays:
-      - name: dcos-1
-        subnet: 192.168.0.0/17
-        prefix: 24
-      - name: dcos-2
-        subnet: 192.168.128.0/17
-        prefix: 24
+      - name: dcos
+        subnet: 9.0.0.0/8
+        prefix: 26
 ```
 
 **Note:** Use a recent Linux kernel (3.9 or later) and Docker version 1.11 or later on the agent nodes.
 
-Each overlay network is identified by a canonical `name`. Containers launched on an overlay network will get an IP address from the subnet allocated to the overlay network. To remove the dependency on a global IPAM, the overlay subnet is further split into smaller container subnets. Each of the container subnets is allocated to an agent. The agents can then use a host-local IPAM to allocate IP addresses from their respective container subnets to containers launched on the agent and attached to the given overlay. The `prefix` determines the size of the container subnet (carved from the overlay subnet) allocated to each agent and thus defines the number of agents on which the overlay can run.
+Each overlay network is identified by a canonical `name` (see [limitations](#limitations) for constraints on naming overlay networks). Containers launched on an overlay network will get an IP address from the subnet allocated to the overlay network. To remove the dependency on a global IPAM, the overlay subnet is further split into smaller subnets. Each of the smaller subnets is allocated to an agent. The agents can then use a host-local IPAM to allocate IP addresses from their respective subnets to containers launched on the agent and attached to the given
+overlay. The `prefix` determines the size of the subnet (carved from the overlay subnet) allocated to each agent and thus defines the number of agents on which the overlay can run.
 
-In the configuration above each overlay network is allocated a /17 subnet (in the “subnet” field), which is then divided into /24 container subnets to be used on each host that will be part of the network (in the “prefix” field) as shown:
+In the configuration above each overlay network is allocated a /8 subnet (in the “subnet” field), which is then divided into /26 container subnets to be used on each host that will be part of the network (in the “prefix” field) as shown:
```

Comment:
Maybe we should also add here:
This would result in the Agent being able to host a maximum of 2^5 Mesos containers and 2^5 docker containers. It is important to note that with this specific configuration, if a framework tries to launch more 2^5 tasks on the Mesos containerzier or the Docker containerizer, it will result in a TASK_FAILURE. Please read the [limitations](#limitations) to understand more about this constraint in the system.